admin/conf/application-logger.xml

```diff
@@ -11,7 +11,7 @@
         </rollingPolicy>
 
         <encoder>
-            <pattern>%date [%thread] %-5level %logger{36} - %msg%n%xException{3}</pattern>
+            <pattern>%date [%thread] %-5level %logger{36} - %msg%n%xException</pattern>
```

Comment:
not sure why this was 3 - would it cause disk space issues?  I found the stack traces in the logs pretty useless when truncated so harshly